.github/workflows/release.yml

```diff
@@ -19,23 +19,19 @@ jobs:
         uses: actions/checkout@v4.1.1
         with:
           fetch-depth: 0
-
       - name: Set up Go
         uses: actions/setup-go@v5.0.0
         with:
           go-version: 1.20.x
-
       - name: Prepare
         id: prepare
         run: |
           TAG=${GITHUB_REF#refs/tags/}
           echo tag_name=${TAG} >> $GITHUB_OUTPUT
-
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
+        uses: docker/setup-qemu-action@v3.0.0
       - name: Login to Docker Hub
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3.0.0
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
Do we have the `DOCKERHUB_USERNAME` and  `DOCKERHUB_TOKEN` placed in the Blade secrets?

Reply:
I will handle that once the code review has been completed. In any case, it will be finished before merging into the develop branch.